pkg/testutil/nerdtest/requirements.go

```diff
@@ -145,8 +145,8 @@ var RootLess = &test.Requirement{
 	},
 }
 
-// RootFul marks a test as suitable only for rootful env
-var RootFul = test.Not(RootLess)
+// Rootful marks a test as suitable only for rootful env
+var Rootful = test.Not(Rootless)
```

Comment:
I expect this to be separate commit, as it doesn’t relate to --quiet

Reply:
Done.